sonar-project.properties

```diff
@@ -13,7 +13,7 @@ sonar.links.scm=https://github.com/DEFRA/apha-apps-perms-move-animal-ui
 sonar.links.issue=https://github.com/DEFRA/apha-apps-perms-move-animal-ui/issues
 
 sonar.sources=src/
-sonar.exclusions=src/**/*.test.js,src/server/common/test-helpers/*
+sonar.exclusions=src/**/*.test.js,src/server/common/test-helpers/*,src/client/javascripts/**/*.js
```

Comment:
Looks like this excludes all client-side code from sonarcloud analysis (not just coverage analysis), which is further than we'd ideally go.